File: Leetcode/1313-DecompressRun-LengthEncodedList.cpp
class Solution {
 public:
  auto decompressRLElist(vector<int>& nums) -> vector<int> {
    auto result = vector<int> {};
    for (auto i = 0; i < nums.size(); i += 2) {
      auto const frequency = nums[i], value = nums[i + 1];
      result.insert(end(result), frequency, value);
    }

    return result;
  }
};

File: Leetcode/1313-DecompressRun-LengthEncodedList.py
class Solution:
    def decompressRLElist(self, nums: List[int]) -> List[int]:
        result = []
        for i in range(0, len(nums), 2):
            frequency, value = nums[i], nums[i + 1]
            result += [value] * frequency

        return result
